src/group/managed_group/mod.rs

```diff
@@ -71,6 +76,8 @@ pub struct ManagedGroup<'a> {
     // The AAD that is used for all outgoing handshake messages. The AAD can be set through
     // `set_aad()`.
     aad: Vec<u8>,
+    // Resumption secret store. This is where the resumption secrets are kept in a rollover list.
+    resumption_secret_store: ResumptionSecretStore,
```

Comment:
Should we really keep them here or push them to the key store? Or do you see a need to always have the values around?

Reply:
Since #334 hasn't landed yet, I left it here. We could also so it as apart of #345.